src/.env

```diff
@@ -1,4 +1,4 @@
-PUBKEY_ADMIN=
-PRIVKEY_ADMIN=
-IS_BACKUP=
-P2P_SERVER_IP=
\ No newline at end of file
+PUBKEY_ADMIN=dabd489642ef9999460f59fe84ed7a7861a25d06a316476b9e0720a9645e221e
```

Comment:
v.g.m niet de bedoeling dat dit in github staat?

Reply:
Inderdaad, maar we hebben nog geen keyvault en ik begrijp dat dit nodig is voor de docker-stack @quintaartsen ?